app/models/form_profile.rb

```diff
@@ -77,34 +97,59 @@ def initialize_identity_information(user)
         suffix: user.va_profile.suffix
       },
       date_of_birth: user.birth_date,
-      gender: user.gender
+      gender: user.gender,
+      ssn: user.ssn
     )
   end
 
   def initialize_contact_information(user)
+    address = {
+      street: user.va_profile.address.street,
+      street2: nil,
+      city: user.va_profile.address.city,
+      state: user.va_profile.address.state,
+      postal_code: user.va_profile.address.postal_code,
+      country: user.va_profile.address.country
+    } if user.va_profile.address
     FormContactInformation.new(
-      address: {
-        street: user.va_profile.address.street,
-        street2: nil,
-        city: user.va_profile.address.city,
-        state: user.va_profile.address.state,
-        postal_code: user.va_profile.address.postal_code,
-        country: user.va_profile.address.country
-      },
+      address: address,
```

Comment:
For that matter `user.va_profile` itself can be nil, if the user is either not LOA3'ed or is not in MVI. Not sure if they'll get this far into this app if those conditions apply, but just wanted to raise it as a possibility.